services.yaml

```diff
@@ -104,6 +225,23 @@ paths:
           description: TBD
         401:
           $ref: "#/definitions/UnauthorizedError"
+  /services/{service_id}/deployments:
```

Comment:
what about a flag for "active"?

Reply:
added:
- in: query
          name: active
          description: Show only active deployments
          type: boolean